### ISSUE TITLE
Will CV-CUDA support findCountours operation? [QUESTION]

### ISSUE THREAD

**Issue by MhLiao** [CVCUDA/CV-CUDA#34]
It seems that the current version does not include `findCountours` .
Will CV-CUDA support findCountours operation? If yes, any plans?

**Comment by milesp-nvidia** [comment]
Hey @MhLiao, I think you're making a feature request, right?

**Comment by MhLiao** [comment]
@milesp-nvidia Yes, I saw there is the `findCountours` operator in the early introduction of CV-CUDA but it is not in the current version.

**Comment by mkhadatare** [comment]
Hi @MhLiao,

We are targetting `findcountours` in CV-CUDA Beta release (ETA end of Q1'23). Will keep you posted.

**Comment by MhLiao** [comment]
Hello @mkhadatare，
When will be the CV-CUDA Beta or the `findCountours` operator released?

**Comment by bhaefnerNV** [comment]
@MhLiao, thank you for your interest in CVCUDA and asking for supporting a FindContours operator. This operator was added [CV-CUDA Release 0.4.0](https://github.com/CVCUDA/CV-CUDA/releases/tag/v0.4.0-beta). Feel free to upgrade CV-CUDA to at least that release to access the operator.
